contexts/linked-vp-v1.jsonld

```diff
@@ -0,0 +1,9 @@
+{
+  "@context": [
+    {
+      "@version": 1.1,
+      "@protected": true,
+      "LinkedVerifiablePresentations": "https://identity.foundation/contexts/linked-vp/v1#LinkedVerifiablePresentations"
```

Comment:
I will go look at some examples but having the "s" on the end feels strange to me if it can sometimes be a single. I often see in JSON-LD land the opposite (having no "s" but allowing more than one of the thing)

Reply:
I followed the example of LinkedDomains - https://www.w3.org/TR/did-spec-registries/#service-types. However, I'm happy to change it to singular.